### ISSUE TITLE
Converter for SGDClassifier looks for deprecated 'log' value.

### ISSUE THREAD

**Issue by Viktor-Nordell** [onnx/sklearn-onnx#908]
The logic in `convert_sklearn_sgd_classifier` checks only for the deprecated loss value `log`, and should be adjusted to look for both `log` and `log_loss` to be compatible with newer version of sklearn.

**Comment by xadupre** [comment]
Changes have been merged and will be released in the next release.
